src/components/LeaderBoard/Leaderboard.jsx

```diff
@@ -279,7 +282,7 @@ const LeaderBoard = ({
                     }
                   </div>
                   {
-                    isAdmin && item.hasSummary && 
+                    hasSummaryIndicatorPermission && item.hasSummary && 
```

Comment:
the variables hasSummaryIndicatorPermission and hasVisibilityIconPermission are used multiple times. To improve code readability and maintainability, I suggest considering grouping the definitions and uses of these variables together or using more descriptive variable names.